ruby/rollbar.rb

```diff
@@ -0,0 +1,10 @@
+Rollbar.configure do |config|
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Ik zou in de default niet de kans willen lopen dat we `access_token` op `nil` zetten. Ik zou hier `ENV.fetch(…)` gebruiken. Voor `environment` zou ik dus `ENV.fetch('ROLLBAR_ENV' Rails.env)` doen.